foam_tokenwrapper/tests/pitzDaily/0/R

```diff
@@ -0,0 +1,52 @@
+/*--------------------------------*- C++ -*----------------------------------*\
```

Comment:
I think that these files are part of the openfoam distribution (https://openfoamwiki.net/index.php/PitzDaily) and they do not need to be part of the reposittory  it should be enough to provide a set of urls and instructions on where to find/download and install them in order to  setup the testing infrastructure.